lib/data_hygiene/publishing_api_sync_check.rb

```diff
@@ -57,6 +59,8 @@ def initialize(scope)
           Whitehall.url_maker.polymorphic_path(model)
         end
       end
+
+      add_translations_expectation
```

Comment:
`PublishingApiSyncCheck` applies to enumerations of content, which may or may not have translations.

This expectation is a no-op for records without translations (when `translation_locales_for(record).empty?`).